ui/options_general.ui

```diff
@@ -213,24 +213,6 @@
           <property name="editable">
            <bool>false</bool>
           </property>
-          <property name="currentIndex">
-           <number>0</number>
-          </property>
-          <item>
-           <property name="text">
-            <string>Stable releases only</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>Stable and Beta releases</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>Stable, Beta and Dev releases</string>
-           </property>
-          </item>
          </widget>
```

Comment:
Re-run `python setup.py build_ui` and commit changes to generated files.

Reply:
Sorry, I forgot all about that.  Done now.